server/app/services/openapi/v2/Swagger2SchemaGenerator.java

```diff
@@ -0,0 +1,167 @@
+              .info(
+                  new Info()
+                      .title(programDefinition.adminName())
+                      .version(Long.toString(programDefinition.id()))
+                      .description(programDefinition.adminDescription())
+                      .contact(
+                          new Contact()
+                              .name("CiviForm Technical Support")
+                              .email(openApiSchemaSettings.itEmailAddress())))
+              .schemes(getSchemes())
+              .security(
+                  new SecurityRequirement().requirement(AuthorizationType.BASIC_AUTH.toValue()))
+              .securityDefinition(AuthorizationType.BASIC_AUTH.toValue(), new BasicAuthDefinition())
+              .path(
+                  "/applications",
+                  new Path()
+                      .get(
+                          new Operation()
+                              .operationId("list_applications")
+                              .description("List Applications")
+                              .summary("Export applications")
+                              .produces(MimeType.Json.getCode())
+                              .tag("programs")
+                              .response(
+                                  200,
+                                  new Response()
+                                      .description("For valid requests.")
+                                      .responseSchema(new RefModel("#/definitions/result"))
+                                      .header(
+                                          "x-next",
+                                          new StringProperty()
+                                              .description("A link to the next page of responses")))
+                              .response(
+                                  400,
+                                  new Response()
+                                      .description(
+                                          "Returned if any request parameters fail validation."))
+                              .response(
+                                  401,
+                                  new Response()
+                                      .description(
+                                          "Returned if the API key is invalid or does not have"
+                                              + " access to the program."))
+                              .parameter(
+                                  new QueryParameter()
+                                      .name("fromDate")
+                                      .type(DefinitionType.STRING.toString())
+                                      .description(
+                                          "An ISO-8601 formatted date (i.e. YYYY-MM-DD). Limits"
+                                              + " results to applications submitted on or after the"
+                                              + " provided date."))
```

Comment:
worth noting that it's in the timezone of the civiform instance?

Reply:
Oh yes. You must have update that on the docs site since I originally wrote that